doc/mgmt/SONiC_YANG_Model_Guidelines.md

```diff
@@ -44,7 +45,7 @@ sonic-vlan.yang
 
 Example :
 ####  YANG
-```
+```yang
```

Comment:
why add yang here?

Reply:
This is to note the below code belongs to YANG type